float64 heading_desired
uint32 KP
uint32 KD
uint16 pwm_max
uint16 pwm_neutral
uint16 rosrate
